common/pom.xml

```diff
@@ -427,5 +411,250 @@
             <groupId>com.googlecode.htmlcompressor</groupId>
             <artifactId>htmlcompressor</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.javascript</groupId>
+            <artifactId>closure-compiler-unshaded</artifactId>
```

Comment:
Same as above. This is defined in line 1313. Add exclusions there.

Reply:
If I move this to master pom, enforcer fails . the other changes were added correctly

Reply:
originally we had that dependency in commons pom, If I remove it completelly, the build fails as well.

Reply:
Ok but we should remove version and scope. You could have added exclusions in the dependency that was defined above already.

Reply:
done!